src/modules/edit-task.js

```diff
@@ -0,0 +1,13 @@
+import getData from './get-data.js';
+
+// Functions
+const editTask = (index, description = '', completed = false) => {
+  if (description) {
+    const toDoTasks = getData();
+    toDoTasks[index - 1].description = description;
+    toDoTasks[index - 1].completed = completed;
+    localStorage.setItem('toDoData', JSON.stringify(toDoTasks));
+  }
+};
```

Comment:
- [ ] Kindly check your local storage because after editing and refreshing the page the updated todo vanished.

Reply:
Thank you @omarsalem7,
I've just applied all the required changes. Thank you for your feedback. 🙂